src/test/client.js

```diff
@@ -370,42 +375,42 @@ function runTest() {
 
     simulateConnection(
         './src/test/dumps/google-standard-stats-p2p',
-        './src/test/results/google-standard-stats-p2p-result.json',
+        './src/test/jest/results/google-standard-stats-p2p-result.json',
```

Comment:
Do we want to switch these test over to use one common results set?
Or should we abandon these tests here and switch to only jest?

Reply:
So this change switches them to the common result set, however the purpose for this test suite was to be more end to end. In a future refactor I'd suggest we remove testing processing results all together here (maybe keep one or two)  and do some more advanced end to end testing, like running multiple connections at the same time, failing the connection, testing the reconnect mechanism once we add and so on, wdyt?